packages/document/docs/en/guide/usage/bundle-size.mdx

```diff
@@ -128,6 +128,49 @@ The **Modules** tag is shown in the right image, from left to right representing
 - **Module Explorer** tag: Click to open the dependency analysis page between `Modules`.
 - **Code View** tag: Click to expand code segments, including `Source` (source code), `Transformed` (compiled code), and `Bundled` (bundled code).
 
+### Support for concatenated module analysis
+
+First, **[Concatenated Module](https://webpack.js.org/plugins/module-concatenation-plugin/#root)** refers to multiple modules that are merged into a single closure, which cannot be analyzed through AST syntax analysis. However, concatenated modules may contain code from different packages, making the analysis of sub-modules within concatenated modules a key focus, especially for projects using the ["all-in-one"](https://rsbuild.rs/guide/optimization/code-splitting#all-in-one) bundling approach.
```

Comment:
Can we link to the Rspack website (If Rspack does not provide the same content as webpack, we can migrate it to Rspack document)